ui/lib/server/widget/message.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class Message extends StatelessWidget {
+  const Message({required this.message});
+
+  final Map message;
+
+  @override
+  Widget build(BuildContext context) {
+    var _list = message.values.toList();
+    return ListView.builder(
```

Comment:
Replace with _Column_ I assume that the message itself isn't supposed to be scrollable, but rather stack content on top of itself